wiki/en/en-Installation-for-Linux.md

```diff
@@ -64,7 +64,7 @@ sudo apt-get install build-essential qt5-qmake qtdeclarative5-dev qt5-default qt
 On **Fedora**:
 
 ```shell
-sudo dnf install qt5-qtdeclarative-devel jack-audio-connection-kit-dbus libQt5Concurrent5 jack-audio-connection-kit-devel
+sudo dnf install qt5-qtdeclarative-devel jack-audio-connection-kit-dbus qt5-qtbase jack-audio-connection-kit-devel
```

Comment:
Doesn't `aqtinstall` work for Qt on Fedora (and most platforms except Windows)?

Reply:
I have only searched for a replacement package in Fedora's standard repos. `aqtinstall` seems not to be in there, but `qt5-qtbase` is.

Reply:
See the autobuild script for MacOS:
https://github.com/jamulussoftware/jamulus/blob/master/autobuild/mac/artifacts/autobuild_mac_1_prepare.sh

Reply:
We use it for the MacOS build -- I wondered if it would save platform-specifics having to be written for the other platforms but it seems the autobuild is very platform specific still...

https://github.com/jamulussoftware/jamulus/blob/master/autobuild/mac/artifacts/autobuild_mac_1_prepare.sh

Reply:
> Doesn't `aqtinstall` work for Qt on Fedora (and most platforms (edit)maybe(/edit) except Windows)?

I can't comment regarding Mac, but I don't think we should require additional packets or scripts (?) on Linux where dependencies can be installed by the native package managers (apt, dnf, yum, etc.). At least it should be like this for users building from source and distributors. As such, it would be the best check for us to do the same in autobuild.

Reply:
I always thought `aqtinstall` was just a way of being able to install a specific version of Qt into a build environment _for Mac only_, so that the installation package produced would contain the version of Qt runtimes required for backward compatibility. For Fedora/CentOS/Debian/Ubuntu, there is no need for `aqtinstall`, because we just use the packaged version of Qt provided by the distro.